mmv1/products/alloydb/Cluster.yaml

```
@@ -62,6 +62,23 @@ examples:
     primary_resource_id: 'full'
     vars:
       alloydb_cluster_name: 'alloydb-cluster-full'
+  - !ruby/object:Provider::Terraform::Examples
+    name: 'alloydb_cluster_restore'
+    primary_resource_id: 'source'
```

```suggestion
    primary_resource_id: 'source'
    skip_test: true
```
This will let the example be added to documentation without generating the unneeded test downstream